src/rpdk/core/contract/suite/handler_delete.py

```diff
@@ -92,7 +100,9 @@ def contract_delete_create(resource_client, deleted_resource):
             response["resourceModel"], resource_client.read_only_paths
         )
 
-        assert deleted_model == response["resourceModel"]
+        assert (
+            deleted_model == response["resourceModel"]
+        ), "A create handler MUST be idempotent"
```

Comment:
Idempotent isn't the right word here (idempotent is used in the context of multiple of the same type of calls not for two separate api calls). This is for the line in the contract that states: "Once a delete operation successfully completes, a subsequent create request with the same primaryIdentifier or additionalIdentifiers MUST NOT return FAILED with an AlreadyExists error code"